reconcile/openshift_base.py

```diff
@@ -609,6 +610,7 @@ def should_apply(
         return False
 
     logging.debug("CURRENT: " + OR.serialize(OR.canonicalize(current.body)))
+    logging.debug("DESIRED: " + OR.serialize(OR.canonicalize(desired.body)))
```

Comment:
nit: i wonder if we should check if debug is enabled on the logger to not seriable and canonicalize when unnecessary

Reply:
I have added a condition to check if debug level logging is enabled